app/models/agents/google_calendar_publish_agent.rb

```diff
@@ -93,8 +102,15 @@ def receive(incoming_events)
      incoming_events.each do |event|
         calendar = GoogleCalendar.new(options, Rails.logger)
 
-        calendar_event = JSON.parse(calendar.publish_as(options['calendar_id'], event.payload["message"]).response.body)
-  
+        # publish using quickadd or insert; default insert
+        if event.payload.key?("method") && event.payload["method"] == "quickadd"
```

Comment:
It'd be cool to also allow the user to set this Agent to always use "quickadd" based on an option.  Something like:

```ruby
if event.payload["method"] == "quickadd" || interpolated["method"] == "quickadd"
```